docs/3.0.0-beta.x/guides/jwt-validation.md

```diff
@@ -17,6 +17,10 @@ When you sign in with the authentication route `POST /auth/local`, Strapi genera
 
 These users are managed in the application's database and can be managed via the admin dashboard.
 
+::: warning
+Please note: In the code example we use default jwt secret which is used for signing tokens. For security reasons it's recommended to set custom secret in `./your-app/extensions/users-permissions/config/jwt.json` or preferably via environment variable `JWT_SECRET`
```

Comment:
Thanks for the PR. This is very important documentation update, based on mongodb-installation document I would suggest following change. (I am also rewording the message to make it more verbose)

::: warning SECURITY TIP
Please note: `users-permissions` extension provides the secret (a string) required to sign the JWT. We strongly recommend that you change this secret in `./your-app/extensions/users-permissions/config/jwt.json`. 

To avoid publishing the secret to your code repository you can pass the secret as an environment variable. To do this change `jwt.json` to...
```
{
  "jwtSecret": "${process.env.JWT_SECRET}"
}
```

Now you can pass the `JWT_SECRET` as...
```
JWT_SECRET=myNewS3c4et npm run develop
```

For more information about using environment variables, please read [dynamic configurations](../concepts/configurations.md#dynamic-configurations).

Reply:
I'd maybe leave it as it is because rewriting it this way would require providing `JWT_SECRET` even during development which might be pretty annoying and redundant. The thing is though, not to use the same token on production, the `JWT_SECRET` env variable is already handled by Strapi by default, so it's enough when this variable is provided in the production environment.